README.md

```diff
@@ -45,3 +45,32 @@ Example response:
 ```
 
 If the user is not found, a 404 status code will be returned.
+
+#### `GET /bans`
+
+Requires `Ban Members` permission
+
+This will return a JSON object with one key, `bans`, which maps to a list of bans, starting with a `user` JSON object, followed by a `reason` JSON object. This mimics discord.py's representation of [BanEntry](https://discordpy.readthedocs.io/en/stable/api.html?highlight=banentry#discord.BanEntry)
+
+Example response:
+```json
+{
+    "bans":[{
+        [
+        "userId": 123456789012345,
+        "userName": "Member",
+        "userDiscriminator": "0000",
+        "userIsBot": False
+        ],
+    "reason": "Banned for Nitro Scams"},
+    {
+    [
+        "userId": 123456789012323,
+        "userName": "MemberTwo",
+        "userDiscriminator": "0001",
+        "userIsBot": False
+        ],
+    "reason": "Banned for Nitro Scams"},
+    ],
+}
```

Comment:
- Please fix indentation and formatting.
- This is not valid JSON, `False` should be `false`.
- `0000` is not a valid discriminator for a user account.

Reply:
Should be corrected in commit BritishBenji@22325ec15730476e0c5f3f468c882503b55bdd1f 
Been too long since I last wrote readable JSON by hand